src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,20 @@
+$secondary-color: #616070;
+$link-color-inactive: #060b35;
+$link-color-active: #00acdc;
+$header-height: 60px;
+$transititon-duraiton-hover: 300ms;
```

Comment:
There is a typo in the variable name `$transititon-duraiton-hover`. It should be corrected to `$transition-duration-hover` to avoid confusion and ensure consistency.